requirements.txt

```diff
@@ -1,5 +1,10 @@
 numpy
 scipy
 matplotlib
-ipywidgets
 git+git://github.com/QENSlibrary/QENSmodels.git
+h5py
+nbsphinx
```

Comment:
Presumably this is just the requirements for using the QENS library, as opposed to building the docs?  If so, do we need to include nbsphinx in here?

Reply:
I removed them.